src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUser } from '../../api';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  choosedTodo: Todo | null;
+  clearChoosedTodo: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  choosedTodo,
+  clearChoosedTodo,
+}) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    getUser(Number(choosedTodo?.userId))
```

Comment:
```suggestion
    getUser(choosedTodo?.userId)
```
why do you need this extra method?